handle/handler.go

```diff
@@ -0,0 +1,24 @@
+package handle
+
+import (
+	"context"
+	"fmt"
+
+	servicebus "github.com/Azure/azure-service-bus-go"
+)
+
+// Handle is a func to handle the message received from a subscription
+type Handle func(ctx context.Context, message, messageType string) Handler
```

Comment:
nit: handle an handler aren't too close for your taste?

Reply:
See other comment